MapboxNavigation/String.swift

```diff
@@ -25,7 +25,9 @@ extension String {
             ("&", "&amp;"),
             ("<", "&lt;"),
             ("\"", "&quot;"),
-            ("'", "&apos;")
+            ("'", "&apos;"),
+            ("(", ""),
+            (")", "")
```

Comment:
Along the lines of osmlab/osmlint#234, are there any other characters that we should strip out?